internal/converter/mlmd_openapi_converter_util.go

```diff
@@ -100,6 +100,21 @@ func MapNameFromOwned(source *string) *string {
 	return &exploded[1]
 }
 
+// MapName derive the entity name from the mlmd fullname
+// for owned entity such as ModelVersion
+// for potentially owned entity such as ModelArtifact
```

Comment:
```suggestion
// for owned entity such as ModelVersion
```

at this point I'd remove the last line, since It's not used by ConvertModelArtifact, as ModelArtifact keeps using the `MapNameFromOwned`